backend/nlp_sent_analysis.py

```diff
@@ -0,0 +1,61 @@
+import string
+
+import matplotlib.pyplot as plt
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import matplotlib.pyplot as plt
+
+print('Libraries imported.')
```

Comment:
generally we don't need to print anything after import statement successfully do what they do